docs/src/main/java/org/geotools/test/FixtureUtilities.java

```diff
@@ -0,0 +1,148 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ * 
+ *    (C) 2005-2010, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+
+package org.geotools.test;
+
+import java.io.BufferedInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.Properties;
+
+import junit.framework.TestCase;
+
+/**
+ * Static methods to support the implementation of tests that use fixture configuration files. See
+ * {@link OnlineTestCase} and {@link OnlineTestSupport} for details.
+ * 
+ * @author Ben Caradoc-Davies, CSIRO Earth Science and Resource Engineering
+ *
+ *
+ * @source $URL$
+ */
+public class FixtureUtilities {
```

Comment:
Not sure why these three test support classes are being added to docs. The docs are just for example code, but perhaps it is useful to have a test case to confirm functionality?

These classes (FixtureUtilites, OnlineTestCase, OnlineTestSupport, TestData) should be available via a maven dependency (rather than duplicated). If we are extra tricky it can be a test dependency...